python/ray/serve/_private/http_proxy.py

```diff
@@ -243,6 +243,9 @@ def get_handle(name):
                 "status_code",
             ),
         )
+        self._ongoing_requests = 0
+        self._ongoing_requests_dummy_obj_ref = None
+        self.active = True
```

Comment:
`self.active` is a bit inaccurate. It seems to only determine whether a proxy will return `503` errors. But a proxy can have `self.active == False` and still be serving pending requests, which I would consider active.

Could we rename this? Maybe `self.available` or `self.quarantined`?